core/src/main/resources/jenkins/install/platform-plugins.json

```diff
@@ -94,6 +94,14 @@
       { "name": "mailer", "suggested": true }
     ]
   },
+  {
+    "category": "Appearance",
+    "plugins": [
+      { "name": "dark-theme", "suggested": true },
+      { "name": "simple-theme-plugin" },
+      { "name": "login-theme" }
```

Comment:
Both of these plugins are fairly easy to configure unsafely, so I'd rather not we offer them to the least experienced of Jenkins admins.